src/readinglistbar.cpp

```diff
@@ -74,12 +74,12 @@ void ReadingListBar::setupList()
                 icon,
                 QString::fromStdString(bookmark.getTitle()),
                 listWidget);
-            item->setTextAlignment(Qt::TextWordWrap);
+            item->setTextAlignment(Qt::AlignLeft);
```

Comment:
How does it work in practice with respect to wrapping behvaiour?

Reply:
To be honest, without `Qt::TextWordWrap` (or even `Qt::AlignLeft') at all I do not see how this helps. It seems from my testing that text wraps automatically to the next line when it can. How would you propose testing?

Reply:
> To be honest, without `Qt::TextWordWrap` (or even `Qt::AlignLeft') at all I do not see how this helps. It seems from my testing that text wraps automatically to the next line when it can.

Does it apply to Qt 5 as well?

Reply:
Tested with Qt 5.12.8

Reply:
Should I check Qt 6?

Reply:
Setting a long string (that includes spaces) as text for the `QListWidgetItem`, works the same as before, for both Qt 5.12.8 and 6.8 for me.

Reply:
Then I believe that his change is safe. But the commit message must be corrected - `Qt::TextWordWrap` has not been deprecated, rather the type of the parameter of `QListWidgetItem::setTextAlignment()` was changed from `int` to `Qt::Alignment`.

Reply:
Sure, will do that.

Should we keep `Qt::Alignment` or remove it completely? In my testing I did not see a difference.

Reply:
Please test with RTL language content. We will decide based on the outcome of that testing.

Reply:
Works as intended with RTL lang content as well. If you agree, I can proceed to totally remove the line.

Reply:
Go ahead! Less code is better.